client/layout/sidebar/item.jsx

```diff
@@ -89,7 +89,18 @@ export default function SidebarItem( props ) {
 				{ ( showAsExternal || props.forceShowExternalIcon ) && ! sidebarIsCollapsed && (
 					<Gridicon icon="external" size={ 24 } />
 				) }
-				{ props.forceChevronIcon && <Gridicon icon="chevron-right" size={ 18 } /> }
+				{ props.forceChevronIcon && (
+					<svg
```

Comment:
I noticed A4A seems to be using `Icon` and `chevronRightSmall` from `'@wordpress/icons'` :) 
https://github.com/Automattic/wp-calypso/blob/12ce866dd558301043b06dd22cf9b77f16d54fcc/client/layout/sidebar-v2/navigator/navigator-menu-item.tsx#L62

Reply:
Nice catch!